pkg/utils/modutils/mod.go

```diff
@@ -0,0 +1,75 @@
+package modutils
+
+import (
+	"bufio"
+	"bytes"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/rotisserie/eris"
```

Comment:
nit: I feel like we have a lot of libs that do this or similar and am wary of adding another. I think this was to avoid importing from go-utils. If so, might be worth extracting our preferred error lib to its own repo.

Reply:
It's not, and actually I have good reason. I looked into this lib, and it really delivers on the promise of errors in go 1.13. I'm open to removing it, but tbh I'd rather swap this out with my implementation from go-utils